### ISSUE TITLE
Cannot read properties of null (reading 'getValue')

### ISSUE THREAD

**Issue by SabineJakubinek** [Flagsmith/flagsmith-js-client#183]
We're using Flagsmith with Plasmic and SSR.
When running "yarn build" I get the following error:"Generating static pages (0/1)PLASMIC: Encountered error when pre-rendering TweetsProvider: TypeError: Cannot read properties of null (reading 'getValue')"

This is my code, which is mostly from the documentation of Plasmic.
`import { DataProvider, usePlasmicQueryData } from "@plasmicapp/loader-nextjs";
import { useFlags } from "flagsmith/react";

export function TweetsProvider(props: { children: React.ReactNode }) {
  const flags = useFlags(["google-analytics-tracking"]);
  if (flags["google-analytics-tracking"].enabled) {
    console.log("Google Analytics enabled");
  }
  const { children } = props;
  const { data } = usePlasmicQueryData("/tweets", async () => {
    console.log("Fetching tweets...");
    const resp = await fetch(
      "https://studio.plasmic.app/api/v1/demodata/tweets"
    );
    return await resp.json();
  });
  if (!data) {
    return null;
  }
  return (
    <DataProvider name="tweets" data={data}>
      {children}
    </DataProvider>
  );
}`

If I comment the four lines after "export function" everything is working fine. But with them the `console.log("Fetching tweets...");` doesn't appear and instead the error comes up. The `usePlasmicQueryData` wraps `useSWR`.

**Comment by dabeeeenster** [comment]
The team here doesnt have any experience with Plasmic I'm afraid - cc @kyle-ssg in case he has any ideas? 

Is it conceptually similar to NextJS?

**Comment by SabineJakubinek** [comment]
Yes, it's a NextJS application. Plasmic and the app are working fine w/o Flagsmith, but not when we implement it. That's why I'm asking here instead asking the guys at Plasmic.

**Comment by kyle-ssg** [comment]
Hi @SabineJakubinek Could I just check whether you're wrapping your root application with the <FlagsmithProvider component? It seems like the hook doesn't have access to flagsmith which comes from the Provider's context.

I should be able to help debug this if you provide an example repository up.

**Comment by kyle-ssg** [comment]
I've added a Plasmic example [here](https://github.com/Flagsmith/flagsmith-js-examples/tree/main/plasmic). If I'm not mistaken, you just needed to ensure you wrapped the root of the app with the provider, as is the case with any React implementation.

To highlight these tools working together I created an example where you could serve different Plasmic pages based on remote config. I hope this is useful!

Please re-open if this didn't solve your issue.
